client/src/components/streams/StreamList.js

```diff
@@ -11,52 +11,58 @@ class StreamList extends React.Component {
   }
 
   renderThumbnail(stream) {
-    var src = null;
+    var src = thumbnails["default"];
+    let iconWidth = "40%";
+    let iconHeight = "60%";
     if (stream.gametag !== undefined) {
       for (var key in thumbnails) {
         if (key === stream.gametag.toLowerCase()) {
           src = thumbnails[key];
+          iconWidth = "100%";
         }
+        console.log(key)
       }
     }
     if (src !== null) {
       return (
+        <span>
         <img
           src={src}
           alt="large middle aligned icon camera"
-          width="30px"
-          height="30px"
+          style={{
+          width:iconWidth,
+          height:iconHeight,
+          }}
         ></img>
+        </span>
       );
-    } else {
-      return <i className="large middle aligned icon camera" />;
     }
   }
 
   renderList() {
+    const alternatingColor = ['#3c3c3c', '#1b1b1b'];
+    
     if (this.props.streams === undefined || this.props.streams.length === 0) {
       return (
         <div>
           <h4 className="white-text">Sorry, no streams available right now.</h4>
         </div>
       );
     } else {
-      return this.props.streams.map((stream) => {
+      return (<div class="stackable three column equal height row"> {this.props.streams.map((stream, index) => {
         return (
-          <div className="item" key={stream.userid}>
-            {this.renderAdmin(stream)}
+          <div className="column center aligned" style={{height:"400px", backgroundColor:alternatingColor[index % alternatingColor.length], padding:"1%"}} key={stream.userid}>
             {this.renderThumbnail(stream)}
-            <div className="content">
+            <div class="center aligned" style={{width:"100%",overflow:"hidden",maxHeight:"3em"}}>
               <Link to={`/streams/${stream.userid}`} className="header">
                 {stream.title}
               </Link>
-              <div className="description white-text">
-                {stream.description}
-              </div>
             </div>
+            <div class="ui fragment description white-text" style={{width:"100%",overflow:"hidden",height:"4em"}}>{stream.description}</div>
```

Comment:
Up to you, but if you add content-scrollable as a class and remove "overflow:"hidden"" from this it lets you scroll the description.